helm/alfresco-process-infrastructure/values.yaml

```diff
@@ -524,9 +524,11 @@ alfresco-modeling-service:
       value: "{{ .Values.content.service.path }}"
     - name: CONTENT_SERVICE_ENABLED
       value: "{{ or .Values.global.acs.enabled .Values.global.hxpr.enabled }}"
-    {{- if .Values.global.hxpr.enabled }}
+    {{- if .Values.global.hxpr.enabled}}
     - name: CONTENT_REPOSITORY_TYPE
       value: "hxp"
+    - name: HXPIDP_ACCOUNTID
```

Comment:
It Is not related to HXPR, so, it shouldn't be in the if statement

Reply:
so could we also change in deployment service
https://github.com/Alfresco/alfresco-process-infrastructure-deployment/blob/develop/helm/alfresco-process-infrastructure/values.yaml#L834C10-L834C10

Reply:
but this Repository is for AAE , so maybe we use that if to check if we are deploying HXP?

Reply:
uhm...I guess that we have to understand if it is HXP or not, because we have only one helm for both. @Giovanni007 can you confirm? thanks

Reply:
yes, checked with @Mieszko96 and we don't have flag like "isHxP" , so we use `hxpr.enabled `to understand that .

Reply:
you can consider create task and add variable ishxp or we can use hxpr.enable